astropy/tests/runner.py

```diff
@@ -156,8 +158,7 @@ def _generate_args(self, **kwargs):
 
         return args
 
-    RUN_TESTS_DOCSTRING = \
-        """
+    RUN_TESTS_DOCSTRING = """
```

Comment:
Is this an API change to the docs?

Reply:
I don't think this will be an API change since thats just fixing the line return.